docs/zh-CN/Packages.md

```diff
@@ -33,62 +34,62 @@ $ ls -l
   completions/hello_world.fish
 ```
 
-通常在 `README.md` 描述插件的工作原理，依据插件情况可以提供一些 [auto completion](http://fishshell.com/docs/current/commands.html#complete)
+いつでも`README.md`にあなたのパッケージがどんな働きをするのかを加えることができます。 また[auto completion](http://fishshell.com/docs/current/commands.html#complete) をよく読んだ上で、適切なタイミングかつ注意した上であなたのユーティリティにそれを加えてください。
 
-`hello_world.fish` 定义一个简单的 function：
+`hello_world.fish`は１つの機能を定義します:
 
 ```fish
 function hello_world -d "Prints hello world"
   echo "Hello World!"
 end
 ```
 
-在你的插件中，每个函数的声明定义必须存放在独立的文件中，这是 fish 自动加载机制强制的，旨在加载功能所需要的，避免无用函数的过渡加载。
+あなたのパッケージの中のどの機能もそれらが記述されたファイル内で宣言されていないといけません。
+オンデマンドで機能をロードしたり、使われない機能がローディングされるのを避けたりするオートローディングのメカニスムによって必須となります。
 
-请记住，fish 缺乏一个私有范围，所以如果你需要分离你的插件到函数，考虑私有函数增加前缀类似 `__hello_world.my_extra_function` 以避免类名冲突和全局命名控件的污染。
+fishはプライベートスコープを持たないということを覚えておいてください。なので、もしあなたのパッケージを機能毎に分割する必要がある場合は次のようにプライベートスコープをプレフィックシングすることを考えてみてください: `__hello_world.my_extra_function`, to avoid both name clashes and すでに存在する値への再定義(global namespace pollution).
 
+# イベント
 
-# 事件
+パッケージはfishイベントを活用するよう設計されました。Oh My Fishがあなたのパッケージに提供する2つのイベントがあります:
 
-利用 fish 事件的优势，目前 Oh My Fish 定义了两个事件提供给开发者使用：
+## 初期化
 
-## 初始化
-
-如果你想在插件被加载时收到[通知](http://fishshell.com/docs/current/commands.html#emit)，你可以在 `hello_world.fish` 添加下面的代码：
+もしもあなたのパッケージのロード中にIf you want to be [notified](http://fishshell.com/docs/current/commands.html#emit), あなたの`hello_world.fish`の中に以下のファンクションを宣言してください:
 
 ```fish
 function init -a path --on-event init_hello_world
   echo "hello_world initialized"
 end
 ```
 
-该事件可以用于修改环境变量、加载资源和自动加载函数等。如果你的插件没有输出任何的函数，你仍然可以使用该事件加载其他的函数。
+環境やロードリソース、オートロード機能などを修正するにはこのイベントを使ってください。もしあなたのパッケージがいかなる機能も出力しなかった場合は, まだパッケージに機能性を加えるためのこのイベントを使うことができます。
 
-## 卸载
+## アンインストール
 
-Oh My Fish 通过 `omf remove <pkg>` 移除已安装的插件前会发送 `uninstall_<pkg>` 事件。订阅者可以使用该事件清理自定义的资源等操作。
+Oh My Fish`omf remove <pkg>`を介してパッケージが取り除かれる前に、`uninstall_<pkg>`イベントを行います。任意でカスタムリソースを整理するためのイベントとして用いることができます。
 
 ```fish
 function uninstall --on-event uninstall_hello_world
 end
 ```
 
 
-# 发布插件
+# 公開
 
-Oh My Fish 在 `$OMF_PATH/db/` 目录注册已发布的插件或主题。
+Oh My Fishでは`$OMF_PATH/db/`以下に公開パッケージのレジストリがあります。
 
-添加你的插件或主题到注册目录需要执行:
+加える必要があるレジストリにあなたのパッケージを加えるには:
 
 ```fish
-# 插件：
+# For packages:
 omf submit pkg/hello_world .../hello_world.git
 
-# 主题：
+# For themes
 omf submit theme/my_theme .../my_theme_name.git
 ```
 
-此操作仅在本地会添加一个新的注册索引，最后你还需要[提交 PR][omf-pulls-link]并入官方的注册索引。
+これはローカルにコピーしたレジストリに対して新しいエントリを加えます。グローバルレジストリにアップデートしたい場合は、[私たちへPRを送る][omf-pulls-link] 必要があります。
 
 
 [omf-pulls-link]: https://github.com/oh-my-fish/oh-my-fish/pulls
```

Comment:
Hello, @MITSUBOSHI 

It looks like you overwrote the Chinese translation in ` docs/zh-CN/Packages.md` instead of creating a new one for Japanese.